modules/services/model/src/main/java/au/com/shiftyjelly/pocketcasts/models/db/dao/UpNextDao.kt

```diff
@@ -33,6 +34,7 @@ abstract class UpNextDao {
         // update the existing episode positions
         val newPosition = updatePositionsBlocking(upNextEpisodes, position, replaceOneEpisode)
         upNextEpisode.position = newPosition
+        upNextEpisode.addedToQueueAt = System.currentTimeMillis()
```

Comment:
I added this new column so we can track when the episode was added to Up next. With this info we can sort back to the original sort

Reply:
I could be wrong but if you add an episode to the top or bottom of your Up Next using row swipe this won't restore the previous to queue order. I think it would be fine to remove the "Added to Up next" option. 

https://github.com/user-attachments/assets/d6f0a1ed-3626-4d33-8819-0ac589b6c7f3

Reply:
@mebarbosa @geekygecko  is this value something that we could somehow sync and have available in the `/up_next/list` endpoint?

Otherwise I think it will be impossible to mirror this feature in the Web Player.